packages/terra-application-utility/src/ApplicationMenuUtility.jsx

```diff
@@ -30,6 +30,10 @@ const propTypes = {
    * The accessory element to be displayed next to the title.
    */
   accessory: PropTypes.element,
+  /**
+   * Role attribute on the Application Menu.
```

Comment:
Can we change this to 
```suggestion
   * The role attribute to set on the menu.
```

Reply:
:thumbsup: [updated](https://github.com/cerner/terra-framework/pull/501/commits/95bc33f8a3c45d78146c62cb9d9add2de6f13b71)​​​​​​​​​​​​​
​​​​​​​